cmd/protoc-gen-cpp-tableau-loader/embed/hub.pc.cc

```diff
@@ -306,6 +306,10 @@ bool LoadMessageWithPatch(google::protobuf::Message& msg, const std::string& pat
     }
   }
   if (existed_patch_paths.empty()) {
+      if (options && options->load_mode == LoadMode::kModeOnlyPatch) {
```

Comment:
No need to check `options` nullity, as code **Line 289** in this func has already checked.